dll/win32/shell32/CShellLink.cpp

```diff
@@ -2832,6 +2838,48 @@ BOOL CShellLink::OnInitDialog(HWND hwndDlg, HWND hwndFocus, LPARAM lParam)
             SendMessageW(hRunCombo, CB_SETCURSEL, index, 0);
     }
 
+    BOOL disablecontrols = darwin;
+    if (!darwin)
+    {
+        WCHAR path[2*MAX_PATH];
+        path[0] = UNICODE_NULL;
+        HRESULT hr = GetPath(path, _countof(path), NULL, SLGP_RAWPATH);
+        if (FAILED(hr))
+            hr = GetPath(path, _countof(path), NULL, 0);
+#if DBG
+        if (GetKeyState(VK_CONTROL) < 0)
+            hr = path[0] = S_OK; // Allow inspection of PIDL parsing path
+#endif
```

Comment:
How can we use this debug feature? Should we press Ctrl while opening the properties dialog?

Reply:
Hold it when switching to the Shortcut tab.

I don't expect this to be a common thing to do but there is a problem with the way ROS currently creates shortcuts that I have to investigate later (certainly for control panel items, maybe other pidl targets as well).